lib/_ffi/js/dom.scm

```diff
@@ -7,25 +7,25 @@
 
 ;;;============================================================================
 
-(define (select selector)
+(define (querySelector selector)
   \document.querySelector(`selector))
 
-(define (get id)
-\document.getElementById(`id))
+(define (getElementById id)
+  \document.getElementById(`id))
 
-(define (create-element type)
+(define (createElement type)
   \document.createElement(`type))
 
-(define (set-attribute elem attr val)
+(define (setAttribute elem attr val)
   \(`elem).setAttribute(`attr, `val))
 
-(define (append-child elem child)
+(define (appendChild elem child)
   \(`elem).appendChild(`child))
 
-(define (insert-adjacent-html elem pos html)
+(define (insertAdjacentHTML elem pos html)
   \(`elem).insertAdjacentHTML(`pos, `html))
 
-(define (log msg)
+(define (console.log msg)
   \console.log(`msg))
 
 (define (alert msg)
```

Comment:
Should add `prompt` and `confirm` too.  It would add convenience to allow any number of parameters and handle them the same way as `print`:

```
(define (alert . rest)
  \alert(`(with-output-to-string (lambda () (apply print rest))))))
```

Reply:
A bit better implementation:

```
(define (alert . rest)
  \alert(`(with-output-to-string (lambda () (##print-aux rest)))))
```